specification/search/data-plane/readme.md

```diff
@@ -78,4 +81,40 @@ csharp:
   namespace: Microsoft.Azure.Search
   output-folder: $(csharp-sdks-folder)/Search/DataPlane/Microsoft.Azure.Search/Generated$(search-folder)
   clear-output-folder: true
-```
\ No newline at end of file
+```
+
+
+## Go

```

Comment:
@marstr Are you guys working on a data-plane Go SDK for Azure Search? You should know that our Swagger spec for the data plane is incomplete. I seem to remember discussing this before, and nothing has changed since: https://github.com/Azure/azure-rest-api-specs/issues/1085

Reply:
Got it, I'll take it out. However, please note that the appropriate course of action would be to develop the API Spec somewhere other than the public `master` branch. Mind removing this folder until it's ready for public consumption?